packages/manager/modules/bm-server-components/src/order-private-bandwidth/private-order.controller.js

```diff
@@ -108,6 +93,7 @@ export default class BmServerComponentsOrderPrivateBandwidthCtrl {
         .then((result) => {
           this.model.orderUrl = result.order.url;
         })
+        .catch((error) => this.handleError(error))
```

Comment:
no need to create a new function
```suggestion
        .catch(this.handleError)
```